src/domain/venue/venueDataFields/VenueDataFields.tsx

```diff
@@ -47,6 +47,22 @@ const VenueDataFields: React.FC<{
             'outdoorActivity',
             data.venue?.outdoorActivity || false
           );
+          setFieldValue(
+            'hasToiletNearby',
+            data.venue?.hasToiletNearby || false
+          );
+          setFieldValue(
+            'hasAreaForGroupWork',
+            data.venue?.hasAreaForGroupWork || false
+          );
+          setFieldValue(
+            'hasIndoorPlayingArea',
+            data.venue?.hasIndoorPlayingArea || false
+          );
+          setFieldValue(
+            'hasOutdoorPlayingArea',
+            data.venue?.hasOutdoorPlayingArea || false
+          );
```

Comment:
Now that we have so many of these fields we could refactor this:

```
          (['outdoorActivity', 'hasSnackEatingPlace'] as const).forEach((v) => {
            setFieldValue(v, data.venue?.[v] || false);
          });
```

To avoid repetition

Reply:
I was thinking something like this, but not this neat. Updated!